pdns/dnsdist-lua-actions.cc

```diff
@@ -540,21 +540,28 @@ DNSAction::Action SpoofAction::operator()(DNSQuestion* dq, std::string* ruleresu
   uint16_t qtype = dq->qtype;
   // do we even have a response?
   if (d_cname.empty() &&
-      d_rawResponse.empty() &&
+      d_rawResponses.empty() &&
       d_types.count(qtype) == 0) {
     return Action::None;
   }
 
   vector<ComboAddress> addrs;
+  vector<std::string> rawResponses;
```

Comment:
We could use a vector of pointers to string to avoid copying the content. Not sure we care much.

Reply:
I would if I knew how :)

Reply:
Let's keep that version, it's safer without juggling with pointers :-)